views/admin/layout/sidebar/crmDataManager.cfm

```diff
@@ -1,17 +1,17 @@
 <cfscript>
     subMenuItems = [];
-    if ( hasCmsPermission( "organisationmanager.read" ) ) {
+    if ( hasCmsPermission( permissionKey="crm.organisation.navigate" ) ) {
         subMenuItems.append( {
-              link  = event.buildAdminLink( linkTo="organisationManager" )
+              link  = event.buildAdminLink( objectName="organisation" )
             , title = translateResource( "cms:crmDataManager.organisations" )
-            , active = ReFindNoCase( "\.?organisationManager$", event.getCurrentHandler() )
+            , active = ReFindNoCase( "^admin\.datamanager", event.getCurrentEvent() ) && ( prc.objectName ?: "" ) == "organisation"
```

Comment:
Does this work correctly when viewing/adding/editing an organisation address / link / note?

Reply:
Fixed